src/components/LyricView.tsx

```diff
@@ -19,6 +19,7 @@ import { useParams } from "react-router";
 
 interface LyricViewProps {
   songNumber: number;
+  setTooNarrow: (tooNarrow: boolean) => void;
```

Comment:
Whats this used for? From the code it looks like nothing, so it may not need to be here

Reply:
its just the props interface for `setTooNarrow` since I'm passing it into lyric view, typescript needs to know the shape/type of the prop